packages/components/organisms/f-header/src/components/Navigation.vue

```diff
@@ -206,63 +206,53 @@
                                 "trakEvent": "click",
                                 "category": "engagement",
                                 "action": "header",
-                                "label": "${accountLogout.gtm}"
+                                "label": "${copy.accountLogout.gtm}"
                             }`'
                         class="c-nav-list-link"
                         v-on="isBelowMid ? { blur: closeNav, focus: openNav } : null">
-                        {{ accountLogout.text }}
+                        {{ copy.accountLogout.text }}
                     </a>
                 </li>
+
+                <country-selector
+                    v-if="showCountrySelector"
+                    ref="countrySelector"
+                    :is-below-mid="isBelowMid"
+                    :current-country-key="copy.currentCountryKey"
```

Comment:
Rather than setting these as individual props, I'd probably just pass through the `copy` object so that it's used consistently across components (and it's clear then in the child component that it's coming from `copy.` in the parent)

Reply:
This also matches how you've changed `Navigation.vue` to work as well.